archetypes/catalog/cli-metadata.properties.mustache

```diff
@@ -13,8 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-build-tools.version={{buildtoolsVersion}}
+build-tools.version=2.0.2
+cli.latest.plugin.version={{cliPluginVersion}}
 cli.version={{cliVersion}}
 {{#cliUpdateMessages}}
-cli.{{version}}.message={{message}}
+    cli.{{version}}.message={{message}}
```

Comment:
Is the whitespace intentional?

Reply:
Yes, this is a mustache loop ; thus the indentation.